modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/rebalance/RebalanceUtil.java

```diff
@@ -490,6 +494,17 @@ public static ByteArray stablePartAssignmentsKey(TablePartitionId partId) {
         return new ByteArray(STABLE_ASSIGNMENTS_PREFIX + partId);
     }
 
+    /**
+     * Key that is needed for the rebalance algorithm.
+     *
+     * @param partId Unique identifier of a partition.
+     * @return Key for a partition.
+     * @see <a href="https://github.com/apache/ignite-3/blob/main/modules/table/tech-notes/rebalance.md">Rebalance documentation</a>
```

Comment:
There is no information about this key in this doc. I would change description to something like "Key for the graceful restart in HA mode", also we can add link to IEP

Reply:
I've created a ticket https://issues.apache.org/jira/browse/IGNITE-24144